recipes/stella-cv-fbow/all/test_package/test_package.cpp

```diff
@@ -1,5 +1,10 @@
 #include <fbow/fbow.h>
+#include <fbow/cpu.h>
+#include <iostream>
 
 int main() {
-    fbow::cpu().detect_host();
+    auto features = fbow::cpu();
+    features.detect_host();
+    std::cout << "CPU Vendor String: " << fbow::cpu::get_vendor_string() << '\n';
```

Comment:
It warms my heard to see `<< '\n'` and not `std::endl`.

Reply:
❤️